docs/reference/search/search-your-data/search-across-clusters.asciidoc

```diff
@@ -290,14 +290,22 @@ means the document came from the local cluster.
 [[skip-unavailable-clusters]]
 === Skip unavailable clusters
 
-By default, a {ccs} returns an error if *any* cluster in the request is
-unavailable.
+By default, a {ccs} returns an error if a remote cluster in the request is
+unavailable. You can override this default for a specific remote cluster using
+the <<skip-unavailable,`skip_unavailable`>> cluster setting.
 
-To skip an unavailable cluster during a {ccs}, set the
-<<skip-unavailable,`skip_unavailable`>> cluster setting to `true`.
+If `skip_unavailable` is `true`, a {ccs}:
 
-The following <<cluster-update-settings,cluster update settings>> API request
-changes `cluster_two`'s `skip_unavailable` setting to `true`.
+* Skips the remote cluster if it's unavailable during the search. The response's
+`_cluster.skipped` value contains a count of any skipped clusters.
+
+* Ignores the <<search-partial-responses,`allow_partial_search_results`>>
+parameter and the related `search.default_allow_partial_results` cluster setting
+when searching the remote cluster. This means searches on the remote cluster may
+return partial results.
```

Comment:
Maybe we should better define what we mean by unavailable. Possibly change wording and say that skip_unavailable is used to make a cluster optional, meaning any error coming from it will be ignored, hence partial results may be returned despite allow_partial_results is set to false.